modules/setting-up-cpu-manager.adoc

```diff
@@ -172,24 +183,50 @@ QoS Class:       Guaranteed
 Node-Selectors:  cpumanager=true
 ----
 
-. Verify that the `cgroups` are set up correctly. Get the process ID (PID) of the `pause` process:
+. Verify that a CPU has been exclusively assigned to the pod by running the following command:
++
+[source,terminal]
+----
+# oc describe node --selector='cpumanager=true' | grep -i cpumanager- -B2
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE    NAME                CPU Requests  CPU Limits  Memory Requests  Memory Limits  Age
+cpuman       cpumanager-mlrrz    1 (28%)       1 (28%)     1G (13%)         1G (13%)       27m
+----
+
+. Verify that the `cgroups` are set up correctly. Get the process ID (PID) of the `pause` process by running the following command:
 +
 [source,terminal]
 ----
+# oc debug node/perf-node.example.com
+sh-4.2# systemctl status | grep -B5 pause
```

Comment:
Split into two code blocks so that users can copy/paste the commands. 

```
[source,terminal]
----
# oc debug node/perf-node.example.com
----

[source,terminal]
----
sh-4.2# systemctl status | grep -B5 pause
----
```